docs/_common/jsonnetwarning.mdx

```diff
@@ -1,6 +1,6 @@
 :::warning
 
 Don't save secrets such as API keys, credentials, or personal data directly in Jsonnet code snippets. Jsonnet code snippets used
-for data mapping aren't stored in an encrypted format in Ory Cloud.
+for data mapping aren't stored in an encrypted format in The Ory Network.
```

Comment:
my 0.02$

Always capitalizing `The` looks really weird, as well as always using "The Ory Network". 
IMO using "the Ory Network" in places where its semantically correct and drop "the" when it is not?